XPlat/AzureMessaging/iOS/source/ApiDefinition.cs

```diff
@@ -166,6 +167,9 @@ public partial interface MSNotificationHub
 		[Static, Export("startWithConnectionString:hubName:")]
 		void Start(string connectionString, string hubName);
 
+		[Static, Export("StartWithInstallationManagement:")]
+		void StartWithInstallationManagement(MSInstallationManagementDelegate managementDelegate);
```

Comment:
It might just be better to call this `Start` and have it as a simple overload?

Reply:
Renamed to start